### PR TITLE
[EDX-69] Move REST history API

### PR DESCRIPTION
**Pull request by tbedford**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

* [EDX-54](https://ably.atlassian.net/browse/EDX-54).
* [EDX-69](https://ably.atlassian.net/browse/EDX-69).

## Review

* [History page](https://ably-docs-edx-69-move-r-1r3ige.herokuapp.com/rest/history/)

### CODE REVIEW

File: content/api/versions/v0.8/rest-sdk/history.textile [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+title: History
+meta_description: "Client Library SDK REST API Reference History documentation."
+meta_keywords: "Ably, Ably REST, API Reference, REST SDK, REST interface, REST API, History"
+section: api
```
Comment (m-hulbert) [inline content/api/versions/v0.8/rest-sdk/history.textile:5]: Missing language selector on this one.

**Review by m-hulbert — CHANGES_REQUESTED** [review]
I noticed there's a few broken in-page links in `/rest/history` for all versions that need to be redirected now the API reference has moved out, e.g. `"paginated":#paginated-result` and `"@Message@":#message` then good to go.

File: content/api/versions/v0.8/rest-sdk/history.textile (hunk 1 above)
Comment (tbedford) [reply content/api/versions/v0.8/rest-sdk/history.textile:5]: Added

**Comment by tbedford** [comment]
@m-hulbert - sorry, missed the comment about the broken links - I will check them.

File: content/rest/versions/v0.8/history.textile [hunk 2]
```diff
@@ -107,7 +106,7 @@ The Ably Rest client library provides a straightforward API to retrieve "paginat
 
 h2(#channel-and-presence). Channel & Presence history
 
-Both the "@Channel@":/rest/channels and "@Presence@":/rest/presence objects provide history. The "@Channel@":/rest/channels object provides the history of "@Message@":#message objects published on the channel, whereas the "@Presence@":/rest/presence object provides presence event history of that channel i.e. members entering, updating or leaving the channel as "@PresenceMessage@":#presence-message objects.
+Both the "@Channel@":/rest/channels and "@Presence@":/rest/presence objects provide history. The "@Channel@":/rest/channels object provides the history of "@Message@":/api/rest-sdk/history#message objects published on the channel, whereas the "@Presence@":/rest/presence object provides presence event history of that channel i.e. members entering, updating or leaving the channel as "@PresenceMessage@":/api/rest-sdk/hsitory#presence-message objects.
```
Comment (m-hulbert) [inline content/rest/versions/v0.8/history.textile:109]:
```suggestion
Both the "@Channel@":/rest/channels and "@Presence@":/rest/presence objects provide history. The "@Channel@":/rest/channels object provides the history of "@Message@":/api/rest-sdk/history#message objects published on the channel, whereas the "@Presence@":/rest/presence object provides presence event history of that channel i.e. members entering, updating or leaving the channel as "@PresenceMessage@":/api/rest-sdk/history#presence-message objects.
```

**Review by m-hulbert — CHANGES_REQUESTED** [review]
Minor typo for the link and all good!

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**